### PR TITLE
Enumerable

### PR DESCRIPTION
**Pull request by dieum512**
In this milestone we:

- Use Ruby syntax for basic programming operations.
- Apply Ruby best practices and language style guides in code, and
- Checked and correct some linter errors

Thank you for the time and energy to review this work

### CODE REVIEW

File: spec/my_list.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative './my_enumeable'
```
Comment (Deepakdanger) [inline spec/my_list.rb]: - [ ] Kindly check whether `my_enumeable` is exist or not, there is an `enumerable.rb` file. getting this error
Screenshot from 2023-06-02 18-03-18

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- Website looks professional :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by MMhaidari** [comment]
# what is added 
- [ ] mentioned error have been solved the code now work properly

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Bravo team, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- You used GitHub flow correctly. ✔
- Well done implementing the requested the change by the previous. 👏 ✔
- You successfully completed all the requirements.  🎉 🎉 🎉 ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**